apps/mobile/mobile-app/components/ui/table/index.web.tsx

```
@@ -1,150 +1,150 @@
+	isHeaderRow: false,
+})
 const TableFooterContext = createContext<{
-  isFooterRow: boolean;
+	isFooterRow: boolean
 }>({
-  isFooterRow: false,
-});
+	isFooterRow: false,
+})
 
 const Table = React.forwardRef(({ className, ...props }: any, ref?: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)